xcell/mappers/mapper_ACT_compsept.py

```diff
@@ -0,0 +1,65 @@
+from .mapper_ACT_base import MapperACTBase
+from pixell import enmap, reproject
+import numpy as np
+from scipy.interpolate import interp1d
+
+
+class MapperACTCompSept(MapperACTBase):
+    def __init__(self, config):
+        """
+        config - dict
+        {'file_map':'tilec_single_tile_D56_cmb_map_v1.2.0_joint.fits',
+         'file_mask':'tilec_mask.fits',
+         'file_noise': 'tilec_single_tile_D56_cmb_map_v1.2.0_joint_noise.fits',
+         'beam_file': 'tilec_single_tile_D56_cmb_map_v1.2.0_joint_beam.txt',
+         'mask_name': 'mask_ACTtsz',
+         'nside': 1024,
+         'lmax': 6000}
+        """
+        self._get_ACT_CompSept_defaults(config)
+
+    def _get_ACT_CompSept_defaults(self, config):
+        self._get_defaults(config)
+        self._get_ACT_defaults(config)
+        self.nl_coupled = None
```

Comment:
Why not move this to `_get_ACT_defaults`?

Reply:
Agreed!